criu/config.c

```diff
@@ -650,6 +650,7 @@ int parse_options(int argc, char **argv, bool *usage_error, bool *has_exec_cmd,
 		{ "ms", no_argument, 0, 1054 },
 		BOOL_OPT("track-mem", &opts.track_mem),
 		BOOL_OPT("auto-dedup", &opts.auto_dedup),
+		BOOL_OPT("zero-pages", &opts.zero_pages),
```

Comment:
To run the CI tests with this option, it also needs to be added in [`zdtm.py`](https://github.com/checkpoint-restore/criu/blob/cda1c5c95aaca5affa28daa59d7f25ed64007a78/test/zdtm.py#L2657)

Reply:
I think I've still not fully understood how your testing framework works, but I did my best to add support for `--skip-zero-pages` to it. Please let me know if there's still something missing.